Core.lua

```diff
@@ -1148,7 +1150,7 @@ addon.groupieLDB = LibStub("LibDataBroker-1.1"):NewDataObject(addonName, {
                                 titleFlag = true
                                 tooltip:AddLine(" ")
                                 tooltip:AddLine(lockout.instance, 255, 255, 255, false)
-                                tooltip:AddLine("|cff9E9E9E  Reset : " ..
+                                tooltip:AddLine("|cff9E9E9E  ".. L["Reset"] .. " : " ..
```

Comment:
Check https://github.com/tekkub/wow-globalstrings/blob/master/GlobalStrings/enUS.lua for built-in strings.

```suggestion
                                tooltip:AddLine("|cff9E9E9E  ".. _G.RESET .. " : " ..
```

https://github.com/tekkub/wow-globalstrings/blob/master/GlobalStrings/enUS.lua#L9323

Then `L["Reset"]` can be removed from Localization/*.lua